check/testdata/nil_constant_crash.go

```diff
@@ -0,0 +1,25 @@
+package foo
+
+import "log"
```

Comment:
There's no need for this test file to import `log` - just use a `println` below.